packages/grid/index.ts

```diff
@@ -37,32 +27,32 @@ export const grid = starryComponent<HTMLElement>(function (
  traits: StarryUITrait[]
 ) {
  const { height, width } =
-  traits.find((x) => x.type === 'cellRender') ?? defaultCellDimensionsTrait
+  traits.find((x) => x.type === 'cellDimensions') ?? defaultCellDimensionsTrait
 
  const { cellRender } =
   traits.find((x) => x.type === 'cellRender') ?? defaultCellRenderTrait
 
  return function (config?: StarryUITraitConfig) {
-  let elem = document.createElement(config?.tagName ?? 'div')
+  const elem = document.createElement(config?.tagName ?? 'div')
   applyTraits(elem, traits, Object.assign({}, defaultConfigConfig, config))
 
-  elem = createGrid({
+  const grid = createGrid({
    cellHeight: height,
    cellWidth: width,
    renderCell: cellRender,
   }).container
 
-  return elem
+  return elem.appendChild(grid)
```

Comment:
❌ Don't `return elem.appendChild(grid)`

Instead:

```
elem.appendChild(grid)
return elem
```

Reply:
🟨 Also let's rename grid here since that's already a top level `const` on line 26. We can call it `const gridElement` instead

Reply:
Done

Reply:
@Vikulakav still both issues needed to be fixed:

❌ Don't `return elem.appendChild(grid)`

Instead:

```
elem.appendChild(grid)
return elem
```

and

🟨 Also let's rename grid here since that's already a top level const on line 26. We can call it const gridElement instead

Reply:
Done

Reply:
✅ Perfect